javascript/ql/test/library-tests/CallGraphs/FullTest/strict.js

```diff
@@ -0,0 +1,10 @@
+(function () {
+  if (true) {
+    function foo() {
+      return 3;
+    }
+  }
+  return foo(); // `foo` is not defined, because we are in strict-mode.
```

Comment:
sanity check: this is the only test that changes output due to the extractor fixup, right? The two other tests remain unchanged.

Reply:
Yes.